2020-12-08 00:30:46 [INFO] persisted: Persisted(last_successful_crawls={'ffmpeg': '2020-12-07 23:54:29.700244+00:00', 'golang-defaults': '2020-12-07 23:54:28.704088+00:00', 'python3-defaults': '2020-12-07 23:54:28.580731+00:00', 'rustc': '2020-12-07 23:54:29.165607+00:00', 'zsh': '2020-12-07 23:54:28.858023+00:00'})
2020-12-08 00:30:46 [INFO] config: Config(feed=FeedConfig(base_url='https://release-feeds.zhimingwang.org/', author='release-feeds', aggregate_title='Aggregate software releases'), packages=['ffmpeg', 'golang-defaults', 'python3-defaults', 'rustc', 'zsh'])
2020-12-08 00:30:46 [INFO] GET https://sources.debian.org/api/src/ffmpeg/
2020-12-08 00:30:46 [INFO] GET https://sources.debian.org/api/src/rustc/
2020-12-08 00:30:46 [INFO] GET https://sources.debian.org/api/src/python3-defaults/
2020-12-08 00:30:46 [INFO] GET https://sources.debian.org/api/src/zsh/
2020-12-08 00:30:46 [INFO] golang-defaults 1.15.6: https://golang.org/dl/go1.15.6.src.tar.gz
2020-12-08 00:30:50 [INFO] GET https://sources.debian.org/api/src/rustc/1.48.0+dfsg1-1/debian/watch
2020-12-08 00:30:50 [INFO] GET https://sources.debian.org/api/src/python3-defaults/3.9.0-4/debian/watch
2020-12-08 00:30:50 [INFO] GET https://sources.debian.org/api/src/ffmpeg/7:4.3.1-5/debian/watch
2020-12-08 00:30:50 [INFO] GET https://sources.debian.org/api/src/zsh/5.8-5/debian/watch
2020-12-08 00:30:50 [INFO] GET https://sources.debian.org/data/main/p/python3-defaults/3.9.0-4/debian/watch
2020-12-08 00:30:50 [INFO] GET https://sources.debian.org/data/main/f/ffmpeg/7:4.3.1-5/debian/watch
2020-12-08 00:30:50 [INFO] GET https://sources.debian.org/data/main/z/zsh/5.8-5/debian/watch
2020-12-08 00:30:50 [INFO] GET https://sources.debian.org/data/main/r/rustc/1.48.0+dfsg1-1/debian/watch
2020-12-08 00:30:51 [INFO] zsh 5.8: https://www.zsh.org/pub/zsh-5.8.tar.xz
2020-12-08 00:30:52 [INFO] rustc 1.48.0: https://static.rust-lang.org/dist/rustc-1.48.0-src.tar.xz
2020-12-08 00:30:52 [INFO] ffmpeg 4.3.1: https://ffmpeg.org/releases/ffmpeg-4.3.1.tar.xz
2020-12-08 00:31:11 [WARNING] retrying uscan for python3-defaults (attempt #1)
2020-12-08 00:31:33 [WARNING] retrying uscan for python3-defaults (attempt #2)
2020-12-08 00:31:58 [WARNING] retrying uscan for python3-defaults (attempt #3)
2020-12-08 00:31:58 [ERROR] failed to update entry for python3-defaults: uscan for python3-defaults failed
2020-12-08 00:31:58 [INFO] no updates found
2020-12-08 00:31:58 [ERROR] failed: python3-defaults
